clang/lib/Index/USRGeneration.cpp

```diff
@@ -757,7 +757,8 @@ void USRGenerator::VisitType(QualType T) {
         case BuiltinType::OCLReserveID:
           Out << "@BT@OCLReserveID"; break;
         case BuiltinType::OCLSampler:
-          Out << "@BT@OCLSampler"; break;
+          Out << "@BT@OCLSampler";
+          break;
```

Comment:
Is this expected?

Reply:
No, this is probably the clang-format side effect. I'll fix it.